editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -1034,7 +1034,7 @@ void SpriteFramesEditor::_animation_name_edited() {
 	edited->set_text(0, name);
 
 	EditorUndoRedoManager *undo_redo = EditorUndoRedoManager::get_singleton();
-	undo_redo->create_action(TTR("Rename Animation"), UndoRedo::MERGE_DISABLE, frames.ptr());
+	undo_redo->create_action(TTR("Rename Animation"), UndoRedo::MERGE_DISABLE, EditorNode::get_singleton()->get_edited_scene());
```

Comment:
This is wrong. It forces undo action for edited scene when editing external resources.

Reply:
Ok, I better revert these changes in this PR, to prevent further regressions.